### PR TITLE
Add documentation

### PR DESCRIPTION
**Pull request by digicosmos86**
Added a `README.md` for documentation specific to ArcherTask

### CODE REVIEW

**Comment by RobertGemmaJr** [comment]
What are the other updates you're making outside of the documentation @digicosmos86 ?

File: src/timelines/section.js
Comment (RobertGemmaJr) [inline src/timelines/section.js:1]: What's the difference between a level and a section?

**Review by RobertGemmaJr — APPROVED** [review]
Looks great! Couple questions for my own understanding but otherwise 🚀

File: package.json [hunk 1]
```diff
@@ -60,7 +61,7 @@
     "eslint-plugin-prettier": "^4.2.1",
     "firebase-admin": "^11.0.1",
     "firebase-tools": "^11.0.1",
-    "husky": "^8.0.3",
+    "husky": "^8.0.0",
```
[the remark this answers is not in the document]
Comment (digicosmos86) [reply package.json:64]: Didn't know it was downgraded. Will fix this

File: src/timelines/section.js
Comment (digicosmos86) [reply src/timelines/section.js:1]: Each level has a separate overlord. A level contains many sections, a section contains many waves, and each wave is a response-feedback loop

**Pull request merged**